scss/components/shellbar.scss

```diff
@@ -162,20 +163,20 @@ $block: #{$fd-namespace}-shellbar;
         padding-right: 0;
         padding-left: $fd-shellbar-item-spacing;
 
-        @include fd-rtl {        
+        @include fd-rtl {
           padding-right: $fd-shellbar-item-spacing;
           padding-left: 0;
         }
 
         // Show only on desktop, hide on s
         @include fd-screen(s) {
-            display: none;
+            //display: none;
         }
         @include fd-screen(m) {
-            display: none;
+            //display: none;
         }
         @include fd-screen(l) {
-            display: inline-block;
+            //display: inline-block;
```

Comment:
Can we remove the commented display:none ?

Reply:
Done. Removed the whole block.